excellent/functions/functions.go

```diff
@@ -350,47 +338,52 @@ func Abs(env utils.Environment, num types.XNumber) types.XValue {
 //   @(round("notnum", 2)) -> ERROR
 //
 // @function round(num [,places])
-func Round(env utils.Environment, args ...types.XValue) types.XValue {
-	if len(args) < 1 || len(args) > 2 {
-		return types.NewXErrorf("ROUND takes either one or two arguments")
-	}
-
-	num, err := types.ToXNumber(args[0])
-	if err != nil {
-		return types.NewXErrorf("ROUND's first argument must be decimal")
-	}
-
-	round := 0
-	if len(args) == 2 {
-		round, err = types.ToInteger(args[1])
-		if err != nil {
-			return types.NewXErrorf("ROUND's decimal places argument must be integer")
-		}
-	}
-
-	return types.NewXNumber(num.Native().Round(int32(round)))
+func Round(env utils.Environment, num types.XNumber, places int) types.XValue {
+	return types.NewXNumber(num.Native().Round(int32(places)))
 }
 
-// RoundUp rounds `num` up to the nearest integer value, also good at fighting weeds
+// RoundUp rounds `num` up to the nearest integer value. You can optionally pass in the number of decimal places to round to as `places`.
 //
-//   @(round_up(12.141)) -> 13
 //   @(round_up(12)) -> 12
+//   @(round_up(12.141)) -> 13
+//   @(round_up(12.6)) -> 13
+//   @(round_up(12.141, 2)) -> 12.15
```

Comment:
i have a hard time believing anyone is building flows that need to round to specific decimal places.. buf if we want to match all legacy functionality, then we need this